web/js/diceware/random-phrase.js

```diff
@@ -0,0 +1,26 @@
+import * as WordList from './wordlist.js';
+import * as RandomIntegers from './random-integer.js';
+
+export { get };
+
+
+function segmentArray(arr,segmentSize) {
+	segmentSize = Math.max(1,Math.min(+segmentSize||0,arr.length));
```

Comment:
Again, appreciate the robustness here, but... IIUC, this function is an internal-only utility, so we control what gets passed in, right? Do we need to guard against `segmentSize` being a non-number?